src/pages/Account/Venue/DetailsValidationSchema.ts

```diff
@@ -2,13 +2,26 @@ import * as Yup from "yup";
 import "firebase/functions";
 import { VenueInput } from "api/admin";
 import { TemplateType } from "settings";
+import firebase from "firebase/app";
 
 type Question = VenueInput["profileQuestions"][number];
 
 export const validationSchema = Yup.object()
   .shape<VenueInput>({
     name: Yup.string()
       .required("Display name required")
+      .test(
+        "dfsd",
+        "This venue name is already taken",
+        async (val: string) =>
+          !(
+            await firebase
+              .firestore()
+              .collection("venues")
+              .doc(val.replace(/\W/g, ""))
+              .get()
+          ).exists
+      )
```

Comment:
nice 🤘

Reply:
quick gut check on this, is it case insensitive?

Reply:
the venue names are case sensitive, yes. It might be good to spend 30mins to make it case insensitive

Reply:
5 mins are enough